### PR TITLE
ResetTemplates and Populate commands improvements.

### PR DESCRIPTION
**Pull request by ekyna**
Do not prompt for confirmation if not interactive. 
New line after index refresh.

### CODE REVIEW

File: src/Command/ResetTemplatesCommand.php [hunk 1]
```diff
@@ -85,6 +85,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->resetter->resetAllIndexes($deleteByPattern);
         }
 
-        return 0;
+        return;
```
Comment (XWB) [inline src/Command/ResetTemplatesCommand.php]: This change is not good practice and will be deprecated from Symfony 4.4: https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.4.md

File: src/Command/ResetTemplatesCommand.php [hunk 2]
```diff
@@ -68,12 +68,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
         $deleteByPattern = $input->getOption('force-delete');
 
-        if ($deleteByPattern) {
+        if ($input->isInteractive() && $deleteByPattern) {
             $helper = $this->getHelper('question');
             $question = new ConfirmationQuestion('You are going to remove all template indexes. Are you sure?', false);
 
             if (!$helper->ask($input, $output, $question)) {
-                return 1;
+                return;
```
Comment (XWB) [inline src/Command/ResetTemplatesCommand.php]: Same as below.

File: src/Command/ResetTemplatesCommand.php (hunk 1 above)
Comment (ekyna) [reply src/Command/ResetTemplatesCommand.php]: Reverted.

File: src/Command/ResetTemplatesCommand.php (hunk 2 above)
Comment (ekyna) [reply src/Command/ResetTemplatesCommand.php]: Reverted.

**Comment by XWB** [comment]
:+1:

**Pull request merged**